### PR TITLE
holdingpen: display ids for new authors detail

### LINKED ISSUE #2130: holdingpen: display BAI info
For new author additions, the BAI info is not displayed although we can find it if we open the record editor.
(requested by Margaret)

### PR DESCRIPTION
**Pull request by harunurhan**
* Fixes #2130

screen shot 2017-03-31 at 15 08 09

### CODE REVIEW

File: inspirehep/modules/workflows/static/js/inspire_workflows_ui/templates/details.html [hunk 1]
```diff
@@ -208,15 +208,15 @@
...
-                    {{id.value}}
-                  </a>
-                </span>
+              <div ng-switch="id.schema" class="col-md-12">
+                <div ng-switch-when="INSPIRE BAI">
+                  <span class="font-bolder">Author profile:</span>
+                  <a href="https://inspirehep.net/author/profile/{{id.value}}">{{id.value}}</a>
+                </div>
+                <div ng-switch-when="ORCID">
+                  <span class="font-bolder">ORCID:</span>
```
Comment (kaplun) [inline inspirehep/modules/workflows/static/js/inspire_workflows_ui/templates/details.html]: Can you also link the ORCID? to `http://orcid.org/{{id.value}}`.

Comment (harunurhan) [inline inspirehep/modules/workflows/static/js/inspire_workflows_ui/templates/details.html]: Addressed @kaplun

**Comment by kaplun** [comment]
One question though, actually also for usability experts @StellaCh @sefeg @jmartinm. Should enforce external link to be opened in a new tab with `target="_blank"`? I know formally this is not recommended. On the other hand the catalogers risk to move out from the application if they click on an external link (with respect to the holding pen) such as an ORCID or the author profile.

**Comment by kaplun** [comment]
https://css-tricks.com/use-target_blank/#article-header-id-7

**Comment by StellaCh** [comment]
Yes, external links should always be `target="_blank"`, otherwise users get confused and might lose track of what they're working on. 
Indeed, in cases where the user workflow is interrupted by creating new tabs, that's not recommended, but here that's not the case.

**Comment by harunurhan** [comment]
@kaplun @StellaCh I made both of them to open in a new tab, but do we consider `inspirehep.net` as external or internal link?

**Comment by kaplun** [comment]
@harunurhan IMHO we can consider external something as soon as it moves out of the current flow in the holding pen. Since holding pen is actually an application, everything that moves out from the flow shall be considered external link.

**Pull request closed without merging**